spec/jekyll_seo_tag_spec.rb

```diff
@@ -237,11 +237,49 @@
     end
 
     context "with absolute site.logo" do
-      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid") }
+      let(:site) { make_site("logo" => "http://cdn.example.invalid/logo.png", "url" => "http://example.invalid", "author" => "Mr. Foo") }
```

Comment:
Maybe we'll save it for when we refactor the tests to bring our coverage back up.